.github/workflows/concrete_compiler_test_cpu_distributed.yml

```diff
@@ -25,12 +24,10 @@ concurrency:
 jobs:
   setup-instance:
     runs-on: ubuntu-latest
-    outputs:
-      runner-name: ${{ steps.start-instance.outputs.label }}
```

Comment:
Is that better to output the runner-name and use it in the rest of the workflow instead of putting `distribued-ci` in hard ?

Reply:
Up to you, it's just to mark the fact that this is persistent runner by looking at the code.